commitizen/changelog.py

```diff
@@ -281,3 +281,49 @@ def incremental_build(new_content: str, lines: List, metadata: Dict) -> List:
     if not isinstance(latest_version_position, int):
         output_lines.append(new_content)
     return output_lines
+
+
+def get_tag_range(tags: List[GitTag], start: str, end: Optional[str]) -> List[GitTag]:
+    accumulator = []
+    keep = False
+    if not end:
+        end = start
+    for index, tag in enumerate(tags):
+        if tag.name == start:
+            keep = True
+        if keep:
+            accumulator.append(tag)
+        if tag.name == end:
+            keep = False
+            try:
+                accumulator.append(tags[index + 1])
+            except IndexError:
+                pass
+            break
+    return accumulator
+
+
+def get_star_and_end_rev(
```

Comment:
We already have a similar function here https://github.com/commitizen-tools/commitizen/blob/master/commitizen/git.py#L71 Do you think we should extend that one instead?

Reply:
I've checked, and they seem to be similar but they are fundamentally:
- one is performing a git command, the other is operating with `List[GitTag]` objects, no command performed.
- one receives start and end, the other works with a string

I think it's better to keep them separated. I've updated the PR to use the same naming.
What do you think?

Reply:
Sounds good to me 👍 But it seems you have a typo here `get_star_and_end_rev` -> get_start_and_end_rev